src/main/webapp/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```

Comment:
Perhaps "Trick Tracker" or something instead of portfolio?

Reply:
Oh my mistake! I forgot to change that. I'll take care of that now